INPUT_SCHEMA.json

```diff
@@ -37,6 +37,20 @@
             "type": "string",
             "description": "Optional text to include in the email notification.",
             "editor": "textfield"
+        },
+        "proxy": {
```

Comment:
I would default it to `None`. I'm' not sure what is the default default :) This way it will not break anything in the UI :)

Reply:
I think it defaults to none as well if you don't provide it (`undefined` works fine for `createProxyConfiguration()`), but I'll set it to useApifyProxy false just in case